### ISSUE TITLE
missing Node.js v18.19.0 on nodejs.org

### ISSUE THREAD

**Issue by okuryu** [nodejs/nodejs.org#6148]
### URL:

- https://nodejs.org/en/blog

### Browser Name:

any

### Browser Version:

any

### Operating System:

any

### How to reproduce the issue:

Node.js version 18.19.0 has been released, but it seems that the information about this version has not been updated on nodejs.org.

- https://github.com/nodejs/node/releases/tag/v18.19.0
- https://nodejs.org/en/blog
- https://nodejs.org/en/about/previous-releases

**Comment by UlisesGascon** [comment]
I confirm the bug! 

The blog post entry exist as https://nodejs.org/en/blog/release/v18.19.0 but is not shown in the index, I will have a look.
![Screenshot 2023-11-30 at 11 50 33](https://github.com/nodejs/nodejs.org/assets/5110813/e205672c-2c76-4680-a3ae-860909bf8b8a)

**Comment by UlisesGascon** [comment]
Not sure why is not indexed, I compared the pr to others and the actions logs. 🤔

**Comment by ovflowd** [comment]
This should properly fix it: https://github.com/nodejs/nodejs.org/pull/6153

**Comment by richardlau** [comment]
> This should properly fix it: #6153

Still missing the posts for [18.19.0](https://github.com/nodejs/nodejs.org/pull/6147) and [20.3.0](https://github.com/nodejs/nodejs.org/pull/6152).
![image](https://github.com/nodejs/nodejs.org/assets/5445507/342195ac-fc85-487a-a16b-ad6454d07965)

**Comment by ovflowd** [comment]
> > This should properly fix it: #6153
> 
> Still missing the posts for [18.19.0](https://github.com/nodejs/nodejs.org/pull/6147) and [20.3.0](https://github.com/nodejs/nodejs.org/pull/6152). ![image](https://private-user-images.githubusercontent.com/5445507/287038234-342195ac-fc85-487a-a16b-ad6454d07965.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEzNzA1MjYsIm5iZiI6MTcwMTM3MDIyNiwicGF0aCI6Ii81NDQ1NTA3LzI4NzAzODIzNC0zNDIxOTVhYy1mYzg1LTQ4N2EtYTE2Yi1hZDY0NTRkMDc5NjUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTEzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzExMzBUMTg1MDI2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YWUzZjcyNDZkMmE3ZTMwMDQxZTVmNGI2M2MwM2Q3YjZmMTI5OWNjMTVjYTI5YmFiZjY1ZTJhYWM1MDIzZDc0NCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.-c9kai5mXydixYTjw91xcf6gelkgK2hAeqWoE86yicY)

Yup, I forgot one change, that the pages themselves also need to be invalidated over time.

**Comment by ovflowd** [comment]
Pushed as hot-fix (https://github.com/nodejs/nodejs.org/commit/74c0c5e13e32d9e75f5f825a6a18fe606854680c)...

Since the data is fetch at build-time and I introduced the "revalidate" on the same build, I bet they just fetched the already cached data. I wonder if a redeployment would have been enough.

But it's something we can monitor. And I can always remove the in-page revalidation if #6153 indeed was enough.
